crates/bevy_diagnostic/src/basic_performance_diagnostics_plugin.rs

```diff
@@ -0,0 +1,99 @@
+use crate::{Diagnostic, DiagnosticId, Diagnostics};
+use bevy_app::{prelude::*, PluginGroupBuilder};
+use bevy_core::FrameCount;
+use bevy_ecs::prelude::*;
+use bevy_time::Time;
+
+/// This plugin group will add "frame time" diagnostics to an App, namely:
+/// * [`FrameTimeDiagnosticsPlugin`](crate::FrameTimeDiagnosticsPlugin)
+/// * [`FpsDiagnosticsPlugin`](crate::FpsDiagnosticsPlugin)
+/// * [`FrameCountDiagnosticsPlugin`](crate::FrameCountDiagnosticsPlugin)
+///
```

Comment:
```suggestion
```

Reply:
Has been changed in [6d8ec89](https://github.com/bevyengine/bevy/pull/7780/commits/6d8ec89d885d90b4fd0e017ea246eae218edcbf8)